src/Explorer/Controls/Tabs/TabComponent.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { AccessibleElement } from "../../Controls/AccessibleElement/AccessibleElement";
+import { Pivot, PivotItem } from "@fluentui/react/lib/Pivot";
```

Comment:
```suggestion
import { Pivot, PivotItem } from "@fluentui/react";
```

You should be able to import these from the root module of `@fluentui/react`, which is better since it doesn't rely on the internal file structure of the module.